### ISSUE TITLE
On Moodle 3.7 with Oracle database

### ISSUE THREAD

**Issue by hmgp** [smartlearningdk/moodle-local_augmented_teacher#3]
Hi, I'm on a Moodle 3.7 with Oracle database.
I install the plugin ok. I try to make a merged message, everything works until I press "Send Message".
Then I get this error:

Coding error detected, it must be fixed by a programmer: Url parameters values can not be arrays!

More information about this error
Debug info:
Error code: codingerror
Stack trace:

line 405 of /lib/weblib.php: coding_exception thrown
line 460 of /lib/weblib.php: call to moodle_url->params()
line 40 of /local/augmented_teacher/messageselect.php: call to moodle_url->param()

Any ideas what it is?

Also, I have an activity with a "Expected complete" date active. But I do not get the option to make reminders. Why is that?

Thank you very much for this awesome plugin.
If I get it to work, it will be very useful.
Moodle should have something like this from the start.

**Comment by hmgp** [comment]
Any help would be appreciated. Thank you.

**Comment by antpetri** [comment]
Any luck fixing the issue? Exactly same thing happens to me on Moodle 3.6. The debug info is all the same.

> Hi, I'm on a Moodle 3.7 with Oracle database.
> I install the plugin ok. I try to make a merged message, everything works until I press "Send Message".
> Then I get this error:
> 
> Coding error detected, it must be fixed by a programmer: Url parameters values can not be arrays!
> 
> More information about this error
> Debug info:
> Error code: codingerror
> Stack trace:
> 
> line 405 of /lib/weblib.php: coding_exception thrown
> line 460 of /lib/weblib.php: call to moodle_url->params()
> line 40 of /local/augmented_teacher/messageselect.php: call to moodle_url->param()
> 
> Any ideas what it is?
> 
> Also, I have an activity with a "Expected complete" date active. But I do not get the option to make reminders. Why is that?
> 
> Thank you very much for this awesome plugin.
> If I get it to work, it will be very useful.
> Moodle should have something like this from the start.

**Comment by hmgp** [comment]
Unfortunately no... Never got help, I left the plugin like it was...

**Comment by antpetri** [comment]
Have you tried [this version](https://github.com/smartlearningdk/moodle-local_augmented_teacher)? It works on 3.6, and I am not sure about more recent versions of Moodle, but hopefully too. At least someone with Moodle 3.7 said it worked for them.

**Comment by hmgp** [comment]
Nope, not tried it anymore. It's been a while so I don't recall if this was the version I tested. Sorry...
